include/mbedtls/rsa.h

```diff
@@ -257,6 +257,14 @@ int mbedtls_rsa_import_raw( mbedtls_rsa_context *ctx,
  *                 the RSA context can be used for RSA operations without
  *                 the risk of failure or crash.
  *
+ *                 Alternative implementations need not support all of the
+ *                 above private key derivations; further, they might put
+ *                 constraints on acceptable sizes and values of the imported
+ *                 parameters. In case a failure originating from these
```

Comment:
The first part of this sentence seems to be missing a predicate. Alternatively it could be "in case _of_...".